qemu/README.md

```diff
@@ -0,0 +1,14 @@
+# QEMU Manager
+
+# Installation
+
+```
+docker build -t qemumanager .
+docker run \
+  --name qemumanager \
+  -v /data/windows-server-2012-r2.qcow2:/data/windows-server-2012-r2.qcow2 \
+  -v /dev/kvm:/dev/kvm \
```

Comment:
We could use the `device` flag to avoid giving to much privileges to the container. https://docs.docker.com/engine/reference/run/#/runtime-privilege-and-linux-capabilities